src/Testing/MqttTests.cs

```diff
@@ -101,31 +101,31 @@ public async Task ReceiveD2CMessageWithQosZero()
         [Test]
         public async Task ReceiveAllC2DMessagesWhileDisconnected()
         {
-            // Arrange
+            //Arrange
             var device = new Device(iotHubDeviceConnectionString);
             var sender = new Sender(iotHubConnectionString, deviceId);
-            var firstPayload = Guid.NewGuid().ToString();
-            var secondPayload = Guid.NewGuid().ToString();
             var payloads = new ConcurrentBag<string>();
             Action<MqttApplicationMessageReceivedEventArgs> applicationMessageReceived = (MqttApplicationMessageReceivedEventArgs e) =>
             {
                 payloads.Add(e.ApplicationMessage.ConvertPayloadToString());
-            };
-
-            // Act
+            };

```

Comment:
```suggestion
            // Act & Assert
            };

```

Reply:
I think you had a good point about the act and assert sections not making a sense, but to keep in convetion with the other tests, I thought this combined comment would be a good idea :)

Reply:
Yes, I agree, that's a good point!